CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Changelog for NeoFS Node
 
 ### Fixed
 - Do not require MainNet attributes in "Without MainNet" mode ([#663](https://github.com/nspcc-dev/neofs-node/issues/663)).
+- User can specify only wallet section without node key ([#xxx](xxx))
```

Comment:
is PR num and link going to be actualized?

Reply:
Yep. It is, already.